code/modules/ai/ai_holder.dm

```diff
@@ -15,7 +15,7 @@
 	/// Which `ai_holder` datum to give to the mob when initialized. If `null`, nothing happens.
 	var/ai_holder_type = null
 	var/image/ai_status_image
-
+	var/is_immune_to_fear = FALSE
```

Comment:
Don't do this, make it a trait instead (would require merging my TG port teehee)